src/mapbox/Map.js

```diff
@@ -34,7 +33,7 @@ class Map extends mixin(MBMap) {
   addLayer(layer, beforeId) {
     if (layer instanceof Layer) {
       this.mobilityLayers.push(layer);
-      this.dispatchEvent('change:mobilityLayer');
+      this.fire('change:mobilityLayers');
```

Comment:
The name of custom event should be in src/common/EventType.js file or elsewhere . It will avoid some typo.